src/main/kotlin/no/nav/familie/ba/mottak/config/KafkaConfig.kt

```diff
@@ -19,11 +20,17 @@ class KafkaConfig {
         return DefaultKafkaConsumerFactory(properties.buildConsumerProperties())
     }
 
+    @Bean
+    fun restartingErrorHandler(taskScheduler: TaskScheduler?): RestartingErrorHandler? {
+        return RestartingErrorHandler(taskScheduler!!)
+    }
+
     @Bean
     fun kafkaListenerContainerFactory(consumerFactory: ConsumerFactory<Int, GenericRecord>) : ConcurrentKafkaListenerContainerFactory<Int, GenericRecord> {
         val factory = ConcurrentKafkaListenerContainerFactory<Int, GenericRecord>()
         factory.containerProperties.ackMode = ContainerProperties.AckMode.MANUAL_IMMEDIATE
         factory.consumerFactory = consumerFactory
+        factory.setErrorHandler(CustomKafkaLoggingErrorHandler())
         return factory
```

Comment:
Ikke noe du behøver å endre, men et alternativ:
        return ConcurrentKafkaListenerContainerFactory<Int, GenericRecord>().apply {
            containerProperties.ackMode = ContainerProperties.AckMode.MANUAL_IMMEDIATE
            this.consumerFactory = consumerFactory
            setErrorHandler(CustomKafkaLoggingErrorHandler())
        }